composer/packages/font/src/icons/object.svg

```diff
@@ -1,23 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <!--
- ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
- ~
- ~   Licensed under the Apache License, Version 2.0 (the "License");
- ~   you may not use this file except in compliance with the License.
- ~   You may obtain a copy of the License at
- ~
- ~        http://www.apache.org/licenses/LICENSE-2.0
- ~
- ~   Unless required by applicable law or agreed to in writing, software
- ~   distributed under the License is distributed on an "AS IS" BASIS,
- ~   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- ~   See the License for the specific language governing permissions and
- ~   limitations under the License.
+	~   Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Leave the license header year as it is.

Reply:
Fixed with 70daa247936ba66d234a0d5f37627c47940a7ad7